modernizer-maven-plugin/src/test/java/org/gaul/modernizer_maven_plugin/ModernizerTest.java

```diff
@@ -418,6 +419,8 @@ public void testAllViolations() throws Exception {
             new ClassReader(Java18Violations.class.getName())));
         occurrences.addAll(modernizer.check(
             new ClassReader(Java19Violations.class.getName())));
+        occurrences.addAll(modernizer.check(
```

Comment:
You need to change the `createModernizer` call to 24 above.

Reply:
Done.